src/components/Feed/AddSexNote.tsx

```diff
@@ -18,13 +39,96 @@ const AddSexNote = () => {
                     'aria-label': 'description',
                 }}
             />
-            <div className={styles.buttons}>
-            <FormControlLabel
+          <Input
+            placeholder="With which person or people you had sex?"
+            defaultValue=""
+            className={styles.input}
+            inputProps={{
+              'aria-label': 'description',
+            }}
+          />
+          <Input
+            placeholder="Note (there you can write which poses have been used, inf about place or everything what you want to remember)"
+            defaultValue=""
+            className={styles.input}
+            inputProps={{
+              'aria-label': 'description',
+            }}
+          />
+          <FormGroup row>
```

Comment:
Take out this FormGroup in other function (no other component, function in this component)